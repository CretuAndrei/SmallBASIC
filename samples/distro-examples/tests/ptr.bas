func foo(s)
 foo="foo: " + s
end

sub bar
 ? "in bar"
end

p = @foo
? call(@foo, "called from ptr @foo")
? call(p, "called from ptr p")

pb = @bar
call pb
call @bar

