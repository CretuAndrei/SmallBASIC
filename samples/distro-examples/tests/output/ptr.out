foo: called from ptr @foo
foo: called from ptr p
in bar
in bar
